requirements.txt

```diff
@@ -1,3 +1,4 @@
 mock==2.0.0
+nose==1.3.7
```

Comment:
Do we really need `nose` though? Built-in `unittest` works fine too as mentioned in
https://github.com/plamere/spotipy/blob/8b84300597be027e0f7d8e40b67d1df04250db57/tox.ini#L8

Reply:
I dont think`nose` is needed. I forgot to check the `tox.ini` file to see how tests were run. I just used it as it was what I was familiar with.

Will remove from `requirements.txt` and use `unittest` command for example in new CONTRIBUTING.md